test/widgets/inbox_test.dart

```diff
@@ -385,6 +444,45 @@ void main() {
           checkAppearsUncollapsed(tester, 1, findSectionContent);
         });
 
+        testWidgets('collapse stream section after scroll', (tester) async {
+          await setupVarious(tester);
+
+          final topicFinder = find.text('stream 1 topic 2' ).hitTestable();
+          final listFinder = find.byType(Scrollable);
+
+          // Scroll the [StickyHeaderListView] enough so that
+          // the [_StreamSection] shows a sticky header
+          await dragUntilInvisible(
+              tester, topicFinder, listFinder, const Offset(0, -100));
```

Comment:
Is there a reason for the move step to be different here from in the other test?

If not — well, the difference makes the reader wonder if there's some subtle reason it needs to be that way. So best to avoid raising that question if there isn't. :slightly_smiling_face:

Reply:
I just increased it in here so that there is less probability of exhausting the `maxIteration`, as we're scrolling more than the other test. But I will make it compatible with the other one, as it will definitely confuse future readers.